src/bidders/prebid/adapters/index-exchange.js

```diff
@@ -21,7 +23,7 @@ export class IndexExchange extends BaseAdapter {
 			bids: sizes.map(size => ({
 				bidder: this.bidderName,
 				params: {
-					siteId,
+					siteId: this.recPlacements && context.get('custom.rec') ? this.recPlacements[code] : siteId,
```

Comment:
What about changing context (defined placements) in app repository instead of creating special logic for handling recovery case here?

Reply:
That's a great idea. I gonna handle this in app repository.